src/field-modules/my-logs/components/EditLog.vue

```diff
@@ -113,38 +113,40 @@
     <h4>Quantities</h4>
     <label for="quantity" class="control-label ">Add new or edit existing quantity</label>
     <div class="form-item form-item-name form-group">
+      <!-- To display a placeholder value ONLY when there are no existing quantities, we must add the placeholder with an <option> tag and select it using the :value option -->
       <select
-        :value="(logs[currentLogIndex].quantity.data.length > 0) ? logs[currentLogIndex].quantity.data[logs[currentLogIndex].quantity.data.length -1].measure : ''"
+        :value="(logs[currentLogIndex].quantity.data.length > 0 && logs[currentLogIndex].quantity.data[0].measure) ? logs[currentLogIndex].quantity.data[0].measure : 'Select measure'"
```

Comment:
Line length. The ternary can be broken into multiple lines, see line 99 for an example.